src/Frontend/Core/Js/frontend.js

```diff
@@ -92,32 +93,49 @@ jsFrontend.controls = {
 }
 
 /**
- * Handles the cookieBar
+ * Handles the privacy consent dialog
  */
-jsFrontend.cookieBar = {
```

Comment:
removing this is not BC

Reply:
Fixed in c96374567c5503b16ae091b4121883be50ce9d41